File: yejin/22_12_02_02.py
# 2차원으로 만들기
# 방법1
# num_list가 존재하는 한 계속 while문 돌려서, n개씩 잘라서 붙이기

def solution(num_list, n):
    answer = [] 
    while num_list:
            answer.append(num_list[:n])                  #n개를 잘라서 answer에 붙이기
            num_list=num_list[n:]                        #앞에 잘라서 붙인 n개를 num_list에서 삭제
    return answer

File: seungbeom/22_12_02_02.java
// https://school.programmers.co.kr/learn/courses/30/lessons/120842?language=java

class Solution {
    public int[][] solution(int[] num_list, int n) {
        int[][] answer = new int[num_list.length/n][n];

        int x = 0;
        while ( x < num_list.length ) {
            for (int y = 0; y < n; y++) {
                answer[x/n][y] = num_list[x++];
            }
        }

        return answer;
    }
}